ospo-mindmap/Content/ospomindmap.md

```diff
@@ -58,9 +58,23 @@ development team of the Linux Project
 
 ## 🧩 Responsibilities
 
-### OS Legal Education
+### 📘 Develop and Execute Open Source Strategy
 
-#### Manage Legal Risks and Licenses
+- Open Source community, contributions and leadership
+- Compliance
+- InnerSource
+- Managing inquiries from customers, employees, investors or community members.
+- Risk tolerance
+- M&A, corporate development
+- Software procurement
+- Marketing
+
+### 🔍 Oversee Open Source Compliance
```

Comment:
It may or may not be on OSPO's plate, but we could add here Open Source Due Diligence for acquisitions (or M&A projects).